### PR TITLE
Added styles & Updated Tagline

### LINKED ISSUE #1038: Style standardization
From the [redpen mockups](https://redpen.io/p/vqdf0d6fb857181a19) it seems like we borrowed quite a few of styles from [Foundation site](http://foundation.mozilla.org/style-guide/). Before updating individual component's styling on Pulse (e.g., nav, footer, buttons, links etc), we should have style standardization in place first.

@natalieworth can you give a complete list of what we should include on Pulse's "style guide"?

For example,

- Colours? (Do we copy the entire colour variable set from Foundation site?)
- Typography?
- Link style?
- Button style?
- anything else?

/cc @youriwims

### PR DESCRIPTION
**Pull request by youriwims**
Closes #1038, #1040

Design flag: The styling rules that we're using from the Foundation-site currently state that links within an `h2` element should inherit the heading's font colour. Here, I had to change those rules in order for the `h2`'s link to be `$dark-blue`(This impacts all heading elements btw). Let me know if that's 🆒 @natalieworth 

This pr would also close the style standardization (#1038) ticket since we've imported the bulk of type, variable, and mixin styles.  /cc @mmmavis

### CODE REVIEW

**Comment by natalieworth** [comment]
> Closes #1038, #1040
> 
> Design flag: The styling rules that we're using from the Foundation-site currently state that links within an `h2` element should inherit the heading's font colour. Here, I had to change those rules in order for the `h2`'s link to be `$dark-blue`(This impacts all heading elements btw). Let me know if that's 🆒 @natalieworth
> 
> This pr would also close the style standardization (#1038) ticket since we've imported the bulk of type, variable, and mixin styles. Let me know if this pr takes care of style standardizatio or if more styles need to be added /cc @natalieworth @mmmavis

Hey, thanks @youriwims I'll update that in my overrides. I think it's something we didn't originally think about for the Foundation site design since we didn't have a use case for it. I think it needs to be `$dark-blue` on the Pulse site so it's immediately recognizable as a link.

**Comment by natalieworth** [comment]
@youriwims That looks really great. The type is awesome! I think I may want to increase the spacing a bit around it—but is that something you'd rather do later/ we could adjust the spacing in a new ticket (as I feel the search and help area may affect it too and we'll probably have to adjust spacing for those)

**Comment by youriwims** [comment]
Okay @mmmavis, makes complete sense! I'll make those updates 👍

File: components/footer/footer.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Footer = () => {
     {
       name: `mozilla`,
       link: `https://mozilla.org`,
-      description: (<p>Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>),
+      description: (<p className="dark-theme body-small">Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>),
```
Comment (youriwims) [inline components/footer/footer.jsx]: small fix until the [footer updates](https://github.com/mozilla/network-pulse/issues/1036) are done.

Comment (alanmoo) [reply components/footer/footer.jsx]: Good catch

File: scss/main.scss [hunk 2]
```diff
@@ -14,9 +15,11 @@
 @import '../node_modules/bootstrap/scss/grid.scss';
 @import '../node_modules/react-select/scss/default.scss';
 
-// mixins
+// Foundation Styles
 
-@import 'mixin.scss';
```
Comment (alanmoo) [inline scss/main.scss]: The /foundation/mixins.scss file doesn't seem to contain everything that this file did. Does everything still work properly/can we get rid of the mixin.scss file?

Comment (youriwims) [reply scss/main.scss]: Everything still works properly, this mixin file is the original pulse mixin file. I just moved it up to line 2 ↗️ to separate it from mofo-bootstrap & foundation styles.

Comment (alanmoo) [reply scss/main.scss]: Wow, I totally missed that.

**Pull request merged**